calculator.py

```diff
@@ -8,8 +9,15 @@ def mul():
     y=int(input("Enter the 2nd number: "))
     m=x*y
     print(f"The multiplication is {m}")
+
+def sqrt():
```

Comment:
change the sqrt function name to square_root

Reply:
since we are following the same conversion name is not important